File: python/problem1180.py
n = int(input())
x = list(map(int, input().split()))
I = 0
least = [x[I], 0]
while I < n:
    if x[I] < least[0]:
        least = [x[I], I]
    I += 1

print('Menor valor: %d' % least[0])
print('Posicao: %d' % least[1])

File: java/Problem1180.java
import java.util.Scanner;

public class Problem1180 {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int n = sc.nextInt();

        int[] x = new int[n];

        for (int i = 0; i < x.length; i++) {
            x[i] = sc.nextInt();
        }

        int menor = 0;

        for (int i = 0; i < x.length; i++) {
            if (x[i] < x[menor]){
                menor = i;
            }
        }
        
        System.out.printf("Menor valor: %d\n", x[menor]);
        System.out.printf("Posicao: %d\n", menor);

        sc.close();
    }
}